src/foam/lib/json/ClassReferenceParser.java

```diff
@@ -69,7 +69,8 @@ public PStream parse(PStream ps, ParserContext x) {
 
     String msg = classId + " is not a modelled class.";
     System.err.println(msg);
-    x.set("error", msg);
-    throw new RuntimeException(msg);
+    var ex = new RuntimeException(msg);
```

Comment:
This was the actual issue?

Reply:
yes, the test was expecting the error in the context to be an actual exception